Extensions/Effects/JsExtension.js

```diff
@@ -613,10 +625,11 @@ module.exports = {
         .setDescription('The resolution of the displacement image')
     );
     glitchProperties.set(
-      'animated',
-      new gd.PropertyDescriptor(/* defaultValue= */ 'true')
-        .setLabel(_('Animated (Enable animations)'))
-        .setType('boolean')
+      'animationFrequency',
+      new gd.PropertyDescriptor(/* defaultValue= */ '0')
```

Comment:
The default value should not be 0 I think

Reply:
yes, for every animated filter the default should be for it to animate

Reply:
Thank you! I have fixed it.

And I move animation of `crt-lines` 10 times faster to make it obvious.
Sorry for no gif. (I cannot upload a `gif` which can show the animation clearly and <= 10M).

Reply:
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/32167255/77252902-e8cc9200-6c91-11ea-8357-eb745afec35f.gif)